app/views/CallForPaper/homeForSpeaker.scala.html

```diff
@@ -85,7 +85,7 @@ <h3><i class="fas fa-tasks"></i> @Messages("your.submission")</h3>
                 </div>
 
                 <div class="alert alert-info">
-                  @Messages("proposal.limit", ConferenceDescriptor.maxProposals())
+                  @Messages("proposal.limit", ConferenceDescriptor.maxProposals(), ConferenceDescriptor.ConferenceProposalConfigurations.concernedByCountQuotaRestrictionAndNotHidden.map(pc => Messages(ProposalType.byProposalConfig(pc).simpleLabel)).mkString("|"))
```

Comment:
See below about the why

```suggestion
                  @Messages("proposal.limit", ConferenceDescriptor.maxProposals(), ConferenceDescriptor.ConferenceProposalConfigurations.concernedByCountQuotaRestrictionAndNotHidden.map(pc => Messages(ProposalType.byProposalConfig(pc).simpleLabel)).mkString(", "))
```